programs/bpf_loader/src/syscalls.rs

```diff
@@ -348,7 +348,7 @@ fn translate_slice_inner<'a, T>(
     {
         Err(SyscallError::UnalignedPointer.into())
     } else if len == 0 {
-        Ok(unsafe { from_raw_parts_mut(0x1 as *mut T, len as usize) })
+        Ok(&mut [])
```

Comment:
I suspect this change is breaking the stable-perf ci test:
https://buildkite.com/solana-labs/solana/builds/39465#641bf1cd-feda-4802-afd9-c8fcd6cbc302

Reply:
nevermind, the issue in the size calc